package.json

```diff
@@ -26,6 +26,7 @@
         "blake",
         "shavite",
         "fugue",
-        "sha1"
+        "sha1",
+	"neoscrypt"
```

Comment:
And here.
Thanks @ahmedbodi you rock!

Reply:
@ahmedbodi any progress on this pull request?